.github/workflows/ci.yml

```
@@ -11,10 +11,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)